modules/webbrowser/src/webbrowsermodule.cpp

```diff
@@ -207,8 +207,11 @@ WebBrowserModule::WebBrowserModule(InviwoApplication* app)
     if (!filesystem::fileExists(subProcessExecutable)) {
         throw ModuleInitException("Could not find web helper executable:" + subProcessExecutable);
     }
-    // CefString(&settings.browser_subprocess_path).FromASCII(subProcessExecutable.c_str());
-
+    #ifndef NOT DARWIN  
```

Comment:
Is this correct?

Reply:
Nope, too much switching between cmake and c++